lib/tasks/cleanup.rake

```diff
@@ -0,0 +1,87 @@
+namespace :cleanup do
+  desc 'Merge contributions from a duplicate author profile into the primary profile given two cap_profile_ids'
+  # Use case: a single author has two author rows with publications associated with each.
+  # You want to merge one author into the author, carrying any existing publications but not duplicating them.
+  # This happens when two profiles are created initially because CAP was not able to match the physician information
+  #  to the faculty information until after two profiles were created.  They "merged" them on the CAP side, but the
+  #  publications were not merged on the SUL-PUB side.  This manifests itself as unexpected behavior (missing pubs, etc.)
+  #
+  # This rake task takes in two cap_profile_ids and will merge all of the publications *from* DUPE_CAP_PROFILED_ID's profile *into* PRIMARY_CAP_PROFILE_ID's profile.
+  # It will then deactivate DUPE_CAP_PROFILED_ID's profile (which should now have no publications associated with it) to prevent harvesting into it.
+
+  # bundle exec RAILS_ENV=production PRIMARY_CAP_PROFILE_ID=123 DUPED_CAP_PROFILED_ID=456 rake cleanup:merge_profiles # will merge all publications from cap_profile_id 456 into 123, without duplication
+  # bundle exec RAILS_ENV=production PRIMARY_CAP_PROFILE_ID=123 DUPED_CAP_PROFILED_ID=456 DRY_RUN=true rake cleanup:merge_profiles # a dry run will show totals and IDs but not change the database
+  task :merge_profiles => :environment do |_t, args|
```

Comment:
Why take `args` and then not use them?  This task should not rely on ENV vars, but have two required arguments as part of its signature.

Reply:
is there a nice way to do named args, so you can clearly which id is which (if you invert them for example, you will get a very wrong result)

Reply:
SO is your best friend, e.g.

Reply:
It's not clear that lets you do is named parameters so you can't make a mistake by sending params in the wrong order (i.e. the "primary" and the "duped" being flipped would lead to the exact opposite result of what you want).

Reply:
@peetucket It should be obvious by how you name the params.  For example, part of the `rake -T` output is:
```bash
rake pubmed:pmid_profile_id_import[path_to_report,cap_profile_id]       
```
That is a result of `pubmed.rake` declaring:
```ruby
task :pmid_profile_id_import, [:path_to_report,:cap_profile_id] ...
```

Reply:
changed ENV variables to args